BUILD.md

```diff
@@ -0,0 +1,147 @@
+the Ubuntu platform has received the highest level of quality assurance,
+testing, and support. Additionally, 32-bit Windows development is not supported.
+
+
+## Prerequisites
+
+To build this package, you will need Python (>= 3.7),
+[Conan][] (>= 1.46), and [CMake][] (>= 3.16).
+These instructions assume a basic understanding of those tools.
+
+[Conan]: https://conan.io/downloads.html
+[CMake]: https://cmake.org/download/
+
+Linux developers will commonly have a default Conan [profile][1] that compiles
+with GCC and links with libstdc++.
+If you are linking with libstdc++ (see profile setting `compiler.libcxx`),
+then you will need to choose the `libstdc++11` ABI:
+
+```
+conan profile update settings.compiler.libcxx=libstdc++11 default
+```
+
+We find it necessary to use the x64 native build tools on Windows.
+An easy way to do that is to run the shortcut "x64 Native Tools Command
+Prompt" for the version of Visual Studio that you have installed.
+
+Windows developers must build rippled and its dependencies for the x64
+architecture:
+
+```
+conan profile update settings.compiler.arch=x86_64 default
+```
+
+Visual Studio 2022 is not yet supported.
+This is because rippled is not compatible with [Boost][] versions 1.78 or 1.79,
+but Conan cannot build Boost versions released earlier than them with VS 2022.
+We expect that rippled will be compatible with Boost 1.80, which should be
+released in August 2022.
+Until then, we advise Windows developers to use Visual Studio 2019.
+
+[Boost]: https://www.boost.org/
+
+
+## Branches
+
+For a stable release, choose the `master` branch or one of the tagged releases
+listed on [rippled's GitHub page](https://github.com/ripple/rippled/releases).
+
+```
+git checkout master
+```
+
+To test the latest release candidate, choose the `release` branch.
+
+```
+git checkout release
+```
+
+If you are contributing or want the latest set of untested features,
+then use the `develop` branch instead.
+
+```
+git checkout develop
+```
+
+
+## How to build and test
+
+This is the general structure of the workflow to build and test rippled:
+
+1. Export [our Conan recipe for RocksDB](./external/rocksdb).
+It builds version 6.27.3, which, as of July 8, 2022,
+is not available in [Conan Center](https://conan.io/center/rocksdb).
+1. Create a build directory. You can choose any name you want.
+1. Build and install dependencies using Conan.
+1. Configure CMake and generate the build system.
+1. Build rippled.
+1. Test rippled.
+The executable's exact location depends on your choice of CMake generator.
+
+Here is an example workflow commonly used to build with a single-configuration
+generator (e.g. Unix Makefiles) on Linux or OSX:
+
+```
+conan export external/rocksdb
+mkdir .build
+cd .build
+conan install .. --output-folder . --build missing --settings build_type=Release
+cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
It is [required](https://gitlab.kitware.com/cmake/cmake/-/issues/23109) if you use a relative path, unquoted, in PowerShell. I included it here because it works everywhere.